rest/stack_trace_handler_uinx.go

```diff
@@ -0,0 +1,54 @@
+//go:build !windows
+// +build !windows
+
+/*
+Copyright 2025-Present Couchbase, Inc.
+
+Use of this software is governed by the Business Source License included in
+the file licenses/BSL-Couchbase.txt.  As of the Change Date specified in that
+file, in accordance with the Business Source License, use of this software will
+be governed by the Apache License, Version 2.0, included in the file
+licenses/APL2.txt.
+*/
+
+package rest
+
+import (
+	"context"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/couchbase/sync_gateway/base"
+)
+
+// registerSignalHandlerForStackTrace will register a signal handler to capture stack traces
```

Comment:
Should https://github.com/couchbase/sync_gateway/blob/618bb6aa5d9d9549908d2d99c52b178634af5136/rest/config.go#L2324 be put into the same area / function?

I didn't investigate this but it looks generic-ish.